applications/static/all_schools.json

```diff
@@ -8665,501 +1403,450 @@
+  "United States University",
   "Unity College",
-  "University of Advancing Technology (UAT)",
+  "University at Albany, State University of New York",
+  "University at Buffalo, State University of New York",
+  "University of Advancing Technology",
   "University of Akron",
-  "University of Alabama - Birmingham",
-  "University of Alabama - Huntsville",
-  "University of Alabama - Tuscaloosa",
-  "University of Alanta",
-  "University of Alaska - Anchorage",
-  "University of Alaska - Fairbanks",
-  "University of Alaska - Southeast",
-  "University of Alaska (System)",
-  "University of Arizona",
-  "University of Arkansas at Fayetteville",
+  "University of Alabama at Birmingham",
+  "University of Alaska Anchorage",
+  "University of Alaska Fairbanks",
+  "University of Alaska Southeast",
+  "University of Arkansas",
+  "University of Arkansas - Fort Smith",
   "University of Arkansas at Little Rock",
   "University of Arkansas at Monticello",
   "University of Arkansas at Pine Bluff",
   "University of Arkansas for Medical Sciences",
-  "University of Arkansas (System)",
   "University of Baltimore",
   "University of Bridgeport",
   "University of California, Berkeley",
   "University of California, Davis",
-  "University of California, Hastings College of Law",
+  "University of California, Hastings College of the Law",
   "University of California, Irvine",
   "University of California, Los Angeles",
   "University of California, Merced",
-  "University of California, Oakland",
   "University of California, Riverside",
   "University of California, San Diego",
   "University of California, San Francisco",
   "University of California, Santa Barbara",
   "University of California, Santa Cruz",
-  "University of California System",
   "University of Central Arkansas",
   "University of Central Florida",
   "University of Central Missouri",
   "University of Central Oklahoma",
-  "University of Central Texas",
   "University of Charleston",
-  "University of Charleston South Carolina",
   "University of Chicago",
   "University of Cincinnati",
-  "University of Colorado at Boulder",
-  "University of Colorado at Colorado Springs",
-  "University of Colorado at Denver",
-  "University of Colorado Health Sciences Center",
+  "University of Colorado Boulder",
+  "University of Colorado Colorado Springs",
+  "University of Colorado Denver",
   "University of Connecticut",
-  "University of Connecticut at Avery Point",
-  "University of Connecticut at Hartford",
-  "University of Connecticut at Stamford",
-  "University of Connecticut at Waterbury",
-  "University of Connecticut Health Center",
   "University of Dallas",
   "University of Dayton",
   "University of Delaware",
   "University of Denver",
   "University of Detroit Mercy",
   "University of Dubuque",
   "University of Evansville",
-  "University of Findlay",
   "University of Florida",
-  "University of Georgia (UGA)",
-  "University of Great Falls",
+  "University of Georgia",
```

Comment:
Can you add the (UGA) part back? Someone's bound to try to type UGA and that'll at least make them conform to what most people are putting.